src/reqs.txt

```diff
@@ -0,0 +1,71 @@
+altair==4.2.0
```

Comment:
General advice is to keep here only project-specific requirements, e.g. packages that you are importing.
In this way, someone can easily see what technologies you use in the repo and install them manually. Also, your environment may contain packages that are already unrelated to the app (move to other libs), dev packages (black, mypy), or hardware-specific (cudnn-c11).

Reply:
I added all the dependencies just after creating venv and importing libs that were necessary. Seems like `altair` is part of `streamlit` module, and if I remove it it refuses to start

Reply:
I removed everything except for `parrot` from git and `streamlit`. Works with only these two 👍